### PR TITLE
Make deliverables grounded on incubated work in the CG

### PR DESCRIPTION
**Pull request by csarven**

### CODE REVIEW

File: charter/index.html [hunk 1]
```diff
@@ -302,14 +302,11 @@ <h3>
                                 <li>
                                     <a href="https://openid.net/connect/">OpenID Connect</a>
                                 </li>
-                                <li>
-                                    Access grants using <a href="https://www.w3.org/2022/06/verifiable-credentials-wg-charter.html">W3C Verifiable Credentials</a>
-                                </li>
```
Comment (pchampin) [inline charter/index.html:307]: Why remove this? Even if it had no prior incubation in the CG, it seems to makes sense. And as a matter of fact, @elf-pavlik did present something today during the CG call what involved VCs, IIUC...

File: charter/index.html [hunk 2]
```diff
@@ -302,14 +302,11 @@ <h3>
...
                                 <li>
                                     Identity mechanisms such as <a href="https://www.w3.org/community/webid/">WebID</a> and <a href="https://www.w3.org/2020/12/did-wg-charter.html">DID</a>
                                 </li>
                                 <li>
-                                    Notifications about resource changes
+                                    Notifications about resource changes using <a href="https://solidproject.org/TR/notifications-protocol">Solid Notifications Protocol</a>
```
Comment (pchampin) [inline charter/index.html:309]: +1 to link to prior art whenever possible

File: charter/index.html (hunk 1 above)
Comment (elf-pavlik) [reply charter/index.html:307]: In the data interoperability panel (one of the CG panels), we look at using VCs to represent details of what access the end user delegated to the client (the app), we discussed it as part of the solution for https://github.com/solid/specification/issues/504 but going beyond the server side clients.

Comment (pchampin) [reply charter/index.html:307]: @elf-pavlik  Then the WG homepage (https://beta.w3.org/groups/wg/vc/ or https://www.w3.org/2017/vc/WG/) would be more appropriate than the charter.

Comment (pchampin) [inline charter/index.html:307]:
```suggestion
                                <li>
                                    Access grants using <a href="https://www.w3.org/groups/wg/vc/">W3C Verifiable Credentials</a>
                                </li>
```

**Comment by csarven** [comment]
Updated and approved as per https://github.com/solid/specification/blob/c63ef36dfab2eaf3a1e6de66a9cb90f184b33753/meetings/2023-03-29.md#make-deliverables-grounded-on-incubated-work-in-the-cg

**Review by elf-pavlik — APPROVED** [review]

**Pull request merged**